ansible/configure_container_infra.yml

```diff
@@ -0,0 +1,39 @@
+---
+# Query OpenStack Manila for details aob
```

Comment:
aob?

Reply:
any other business? this was a copy and paste from the README.